lib/senkyoshi/canvas_course.rb

```diff
@@ -153,8 +162,8 @@ def upload_scorm_package(scorm_package, course_id, tmp_name)
     # Creates assignments from all previously uploaded scorm packages
     ##
     def create_scorm_assignments(scorm_packages, course_id, local)
-      scorm_packages.each do |pack|
-        create_scorm_assignment(pack, course_id, local)
+      scorm_packages.map do |pack|
```

Comment:
I looked into it a bit this morning, and I can't quite figure out how to make it work in Minitest. Maybe you have more experience and a better idea of how it could work though. What I am running into is that I would have to mock entire CanvasCourse in order to have any expectation about how many times a particular method in the CanvasCourse was called.